042-SAPOnAzure/Student/04-BusinessContinuity-and-DR.md

```diff
@@ -0,0 +1,92 @@
+# Challenge 4: Business Continuity and Disaster Recovery with Azure NetApp Files
+
+[< Previous Challenge](./03-SAP-Security.md) - **[Home](../README.md)** - [Next Challenge >](./05-PowerApps.md)
+
+## Introduction
+
+Contoso heavily relies on SAP infrastructure to perform day to day business transactions. Unavailability of IT infrastructure for SAP or SAP application itself can heavily impact business transactions and potentially delay revenue recognition. Contoso is concerning about the data consistency on backups and restorability with online backups and necessity of offline backups along with periodicity. CIO heard about Azure netapp files and its features and took a decision to use ANF across the SAP infrastructure for Hana database.  
+
+SAP S/4 Hana system is fully protected with required IT monitoring, secured & compliance configuration and also with high availabitly for IT component failures. However, it is not protected with accidental errors, unintended data discrepencies, data loss / corruption or geographical catastrophies. Design and implement BCDR solution for SAP S/4 Hana system by implementing secondary copy of SAP system in another seperate azure region from production region with continuous asynchronous data replication.
+
+![Backup and Disaster Recovery - HANA on ANF](Images/Challenge4Arch.PNG)
+
+## Description
+
+1. Backup using a permanent solution (ANF snapshots)
+	- The backup team at Contoso has already finished assessing backup requirements and have provided you the below backup schedule `(See Table in Figure 1 below)`
```

Comment:
don't use the backticks here.... you can make the "(See Table..." bold instead or italic.

Reply:
good catch, updated.